updates/35/updates.sql

```diff
@@ -639,4 +639,186 @@ INSERT INTO assets(identifier, name, type) VALUES('SBTC', 'Super Bitcoin', 'W');
+UPDATE common_asset_details SET coingecko = 'everclear', symbol = 'CLEAR' WHERE identifier = 'eip155:137/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8';
+INSERT INTO assets(identifier, name, type) VALUES('eip155:137/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'Everclear', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:137/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'A', 137, '0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:137/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'CLEAR', 'everclear', 'NEXT', NULL, 1693777295, NULL);
+UPDATE common_asset_details SET coingecko = 'everclear', symbol = 'CLEAR' WHERE identifier = 'eip155:56/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8';
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'Everclear', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'A', 56, '0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'CLEAR', 'everclear', 'NEXT', NULL, 1693777295, NULL);
+UPDATE common_asset_details SET coingecko = 'everclear', symbol = 'CLEAR' WHERE identifier = 'eip155:10/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8';
+INSERT INTO assets(identifier, name, type) VALUES('eip155:10/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'Everclear', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:10/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'A', 10, '0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:10/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'CLEAR', 'everclear', 'NEXT', NULL, 1693777295, NULL);
+UPDATE common_asset_details SET swapped_for = 'eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8' WHERE identifier = 'eip155:1/erc20:0xFE67A4450907459c3e1FFf623aA927dD4e28c67a';
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'Everclear', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'A', 1, '0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8', 'CLEAR', 'everclear', 'NEXT', NULL, 1693777295, NULL);
```

Comment:
This looks wrong. Shouldn't you first create `'eip155:1/erc20:0x58b9cB810A68a7f3e1E4f8Cb45D1B9B3c79705E8'` since it isn't in the db and then change the swapped for? The insert statement should be an insert for `0xFE67A4450907459c3e1FFf623aA927dD4e28c67a` with the swapped for field up to date